app/assets/javascripts/modules/inline-image-modal.js

```diff
@@ -32,6 +32,30 @@ InlineImageModal.prototype.fetchModalContent = function (url) {
     })
 }
 
+InlineImageModal.prototype.postModalForm = function (formID) {
+  var form = document.getElementById(formID)
+  var controller = new window.AbortController()
+  setTimeout(function () { controller.abort() }, 10000)
+
+  var options = {
+    credentials: 'include',
+    signal: controller.signal,
+    headers: { 'Content-Publisher-Rendering-Context': 'modal' },
+    redirect: 'follow',
+    method: 'POST',
+    body: (new window.FormData(form))
```

Comment:
again minor, but these brackets look superfluous

Reply:
Did we confirm that using FormData object was sufficiently supported cross-browser?

Reply:
Yup 👍 - even made it an A/C on the card.